src/components/DTTPeriod/DTTPeriod.tsx

```diff
@@ -66,7 +66,7 @@ export const DTTPeriod = ({
       expandedContent={
         <>
           at{" "}
-          {showTimesInsteadOfRooms
+          {!showTimesInsteadOfRooms
```

Comment:
Why do you want `!` so badly

Reply:
It makes more sense logically, since it conveys that whether it is the time or the room is the opposite of the `rightContent`